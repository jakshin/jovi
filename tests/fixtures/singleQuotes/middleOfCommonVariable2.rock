(A possessive doesn't work on a common variable's prefix)
My's answer is 42
Say my answer
