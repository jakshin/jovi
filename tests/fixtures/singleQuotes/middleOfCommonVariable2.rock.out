Unexpected possessive keyword 'My's' here
